Competitive Coding/Linked List/README.md

```diff
@@ -0,0 +1,19 @@
+# Linked List operations
+
+**This folder contains files that have the following algorithms**
+
+ 1. All BASIC Linked List operations on a Linked List based on *Pointers*
+   
+    * Adding elements at the *end* of a Linked List.
+    * Adding elements at the *begining* of a Linked List.
```

Comment:
Like give a link to the subpart of the Linked List programming demonstrating this.

Reply:
Okay I will update the README and send it to you soon